lib/api/model/model.dart

```diff
@@ -302,6 +314,10 @@ enum UserRole{
   final int? apiValue;
 
   int? toJson() => apiValue;
+
+  bool isAtLeast(UserRole role) {
+    return (apiValue ?? 0) <= (role.apiValue ?? 0);
+  }
```

Comment:
> when servers give an API value we don't recognize—say, 350, which would be between 300 "moderator" and 400 "member"—we could still store that value, and use it in the new isAtLeast method.

I think in the current code for `UserRole` we get `UserRole.unknown` for all the unrecognized API values. With that being said, we cannot get that new unrecognized value to compare it with the threshold apiValue. 🙂

Reply:
The proposal is to store the unrecognized API value so that `isAtLeast` can get its hands on it :). Looking closer, I think that would need some preparation, though, such as converting `UserRole` from an `enum` to a plain `class`.

Greg reminds me that we don't actually expect any more API values to be added, though, because the "role" concept is on track to be replaced by "user groups". So the work I've described for handling unrecognized values won't be worth it.